### ISSUE TITLE
When you double-click to edit text, is it possible to open up the text side panel?

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#1216]
When you are creating, when you happen to be on a tab different to the text panel, is it possible to open up the text side panel when you double-click the text box? Because you are obviously wanting to edit the text when you double-click it.

**Comment by dakom** [comment]
Yes but this isn't on the component level, it's logic which has to be added carefully to every specific module editor

**Comment by corinnewo** [comment]
Good - then let's do it, possibly not for MVP, but keep this ticket alive.
